Templates/CSharp/EdgeDriverTemplate/EdgeDriver.xml

```diff
@@ -0,0 +1,8 @@
+<Zips>
```

Comment:
Are the binary files included in this PR (.zips and .nupkgs) included by design? Asking because binary files can bloat git history over time.

Reply:
It is also not clear to me why the zip files for VB and UWP were changed below in this PR.

Reply:
These nuget pacakges are copied manually. The newest master branch has put .zip to .gitignore. 
I reabse the branch, so it won't change the zip file under VB and UWP